service/Microsoft.Azure.Devices/AmqpServiceClient.cs

```diff
@@ -153,7 +164,7 @@ public SendingAmqpLink SendingLink
                 throw AmqpErrorMapper.GetExceptionFromOutcome(outcome);
             }
         }
-
+        
```

Comment:
Nit: please remove the extra white-space on this empty line.

Reply:
Nit: @ryan-z-johnson you may have forgotten to remove this ws :) - or GH has a bug(?)

Reply:
I think it is a bug. The only extra line I have in that area is one line between the two functions